agent/01_agent_requirements.sh

```diff
@@ -1,15 +1,44 @@
 #!/bin/bash
 
-set -o pipefail
+# set -o pipefail
```

Comment:
why removing the pipefail?
I would also add unbound variables check if possible

Reply:
I forget why that was done for debug, fixed.